jetty-core/jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -313,6 +313,8 @@ protected IdentityService findIdentityService()
     protected void doStart()
         throws Exception
     {
+        Context context1 = ContextHandler.getCurrentContext();
```

Comment:
I don't see this referenced anywhere in the code?

Reply:
oops..

removed